inference-engine/tests/functional/plugin/shared/include/single_layer_tests/ctc_loss.hpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <string>
+#include <tuple>
+#include <vector>
+
+#include "functional_test_utils/layer_test_utils.hpp"
+
+
+typedef std::tuple<
+        std::vector<size_t>,               // Logits shapes
+        std::vector<int>,                  // logits lenght
```

Comment:
minor comment
maybe should change ```int``` using in lenght to ```size_t```

Reply:
This type meets specification.